reference_docs/YAML_models/ietf_v3_document_attributes.yaml

```diff
@@ -0,0 +1,583 @@
+    more_information: See https://tools.ietf.org/html/rfc7991#section-2.65[here]
+
+  xml-lang:
+    tags: ["Document info"]
+    title: "XML language"
+    description: "Set the document language. By default this is `en`. RFC XML: `rfc@xml:lang`."
+
+  consensus:
+    tags: ["Document info"]
+    title: "Consensus"
+    description: |
+      Set document consensus for this document. The following values are allowed:
+
+      * `false`
+      * `true`
+
+      RFC XML: `rfc@consensus`.
+    value_spec: boolean
+
+  index-include:
+    tags: ["Document info"]
+    title: "Index include"
+    description: |
+      Specifies whether formatter should include an index in generated files. If the source file has no `<iref>` elements, an index is never generated. +
+      RFC XML: `rfc@indexInclude`
+
+      NOTE: This attribute is new in AsciiRFC v3 (RFC XML v3).
+    value_spec: boolean
+    value_default: true
+
+  sort-refs:
+    tags: ["Document info"]
+    title: "Sort references"
+    description: |
+      Specifies whether the prep tool should sort references. +
+      RFC XML: `rfc@sortRefs`
+
+      NOTE: This attribute is new in AsciiRFC v3 (RFC XML v3).
+    value_spec: boolean
+    value_default: false
+
+  sym-refs:
+    tags: ["Document info"]
+    title: "Symbolic references"
+    description: |
+      Specifies whether formatter should use symbolic references (such as "`[RFC211]`") or not (such as "`[3]`"). +
+      RFC XML: `rfc@symRefs`
+      NOTE: This attribute is new in AsciiRFC v3 (RFC XML v3).
+    value_spec: boolean
+    value_default: true
+
+  toc-include:
+    tags: ["Document info"]
+    title: "ToC include"
+    description: |
+      Specifies whether formatter should contain a table of contents.
+      RFC XML: `rfc@tocInclude`
+      +
+      NOTE: This attribute is new in AsciiRFC v3 (RFC XML v3).
+    value_spec: boolean
+    value_default: true
+
+  toc-depth:
+    tags: ["Document info"]
+    title: "ToC depth"
+    description: |
+      Determines the depth of the table-of-contents; e.g. a value of `3` means three levels of heading are included. RFC XML: `rfc@tocDepth`
+      +
+      NOTE: This attribute is new in AsciiRFC v3 (RFC XML v3).
+
+  show-on-front-page:
+    tags: ["Document info"]
+    title: "Show on front page"
+    description: |
+      Display organization of author on front page of IAB document. Introduced in Levkowetz' implementation notes. RFC XML: `organization/@showOnFrontPage`, applied to all organizations named in the document front matter.
+      +
+      NOTE: This attribute is new in AsciiRFC v3 (RFC XML v3).
+    value_spec: boolean
+    value_default: true
+
+#--- Author attributes ---#
+  fullname:
+    tags: ["Author info"]
+    title: "Fullname"
+    description: 'Author&apos;s full name. Can set here instead of document header&apos;s "`Author`" line. RFC XML: `front/author@fullname`.'
+
+  initials:
```

Comment:
@opoudjis 
in v2 there was an attribute called `forename_initials` which was different from standocs `initials` because it excluded the last name. 
in v3 the tag is now called `initials` like in standoc but it behaves like in v2, and excludes the last name. Should the tag be updated to `forename_initials`?

Reply:
The metanorma standoc initials are not meant to include the surname either, so this is more a fault in the standoc documentation.